hnginternship5/ziki
# Lucid- Drafts Page redesign

#### What does this PR do?
* It creates the drafts page

#### Description of Task to be completed?
* Redesign the drafts page to suit the approved design

#### How should this be manually tested?
* Open the site on your browser
* Type localhost/drafts
* You should see the drafts page, responsive

#### Screenshots (if appropriate)
![Screen Shot 2019-05-01 at 01 28 42](https://user-images.githubusercontent.com/33119163/57001934-68053300-6bb3-11e9-8d08-4a8a462475ce.png)
![Screen Shot 2019-05-01 at 01 28 16](https://user-images.githubusercontent.com/33119163/57001755-55d6c500-6bb2-11e9-8d7b-40aaa6e38aef.png)
![Screen Shot 2019-05-01 at 01 28 25](https://user-images.githubusercontent.com/33119163/57001936-69cef680-6bb3-11e9-8889-0b1b2f022cd0.png)


#### Questions/Comments:
